akka-cluster-typed/src/multi-jvm/scala/akka/cluster/typed/ChunkLargeMessageSpec.scala

```diff
@@ -0,0 +1,216 @@
+  import ChunkLargeMessageSpec._
+
+  // TODO move this to MultiNodeTypedClusterSpec
+  def identify[A](name: String, r: RoleName): ActorRef[A] = {
+    import akka.actor.typed.scaladsl.adapter._
+    val sel = system.actorSelection(node(r) / "user" / "testSpawn" / name)
+    sel.tell(Identify(None), testActor)
+    expectMsgType[ActorIdentity](10.seconds).ref.get.toTyped
+  }
+
+  private def test(n: Int, numberOfMessages: Int, includeLarge: Boolean): Unit = {
+    runOn(first) {
+      val producerController = spawn(ProducerController[Consumer.TheMessage](s"p$n", None), s"producerController$n")
+      val producer =
+        spawn(Producer(numberOfMessages, large = false, delay = 10.millis, producerController), s"producer$n")
+      val largeProducerController =
+        spawn(
+          ProducerController[Consumer.TheMessage](
+            s"p$n",
+            None,
+            ProducerController.Settings(typedSystem).withChunkLargeMessagesBytes(50000)),
+          s"largeProducerController$n")
+      val largeProducer =
+        spawn(
+          Producer(if (includeLarge) Int.MaxValue else 0, large = true, delay = 10.millis, largeProducerController),
+          s"largeProducer$n")
+      enterBarrier(s"producer$n-started")
+      val probe = TestProbe[Any]()
+      probe.expectTerminated(producer, 25.seconds)
+      largeProducer ! Producer.Stop
+      enterBarrier(s"producer$n-stopped")
+    }
+    runOn(second) {
+      enterBarrier(s"producer$n-started")
+      val consumerController = spawn(ConsumerController[Consumer.TheMessage](), s"consumerController$n")
+      val consumer = spawn(Consumer(consumerController), s"consumer$n")
+      val largeConsumerController = spawn(ConsumerController[Consumer.TheMessage](), s"largeConsumerController$n")
+      val largeConsumer = spawn(Consumer(largeConsumerController), s"largeConsumer$n")
+      val producerController: ActorRef[ProducerController.Command[Consumer.TheMessage]] =
+        identify(s"producerController$n", first)
+      consumerController ! ConsumerController.RegisterToProducerController(producerController)
+      val largeProducerController: ActorRef[ProducerController.Command[Consumer.TheMessage]] =
+        identify(s"largeProducerController$n", first)
+      largeConsumerController ! ConsumerController.RegisterToProducerController(largeProducerController)
+      enterBarrier(s"producer$n-stopped")
+      consumer ! Consumer.Stop
+      largeConsumer ! Consumer.Stop
+    }
+    enterBarrier(s"after-$n")
+  }
+
+  "Reliable delivery with chunked messages" must {
+
+    "form a cluster" in {
+      formCluster(first, second)
+      enterBarrier("cluster started")
+    }
+
+    "warmup" in {
+      test(1, 100, includeLarge = true)
+    }
+
+    "measure latency without large messages" in {
+      test(2, 250, includeLarge = false)
+    }
+
+    "measure latency with large messages" in {
+      test(3, 250, includeLarge = true)
+    }
```

Comment:
This test has one pair of actors sending large messages and another pair sending small messages.
It works as expected with the chunking enabled. When it's not enabled the small messages suffer from the large messages.

![Histogram](https://user-images.githubusercontent.com/336161/79153289-e6abae00-7dcd-11ea-9f68-4bbae5d09074.png)